test/test.js

```diff
@@ -950,6 +963,54 @@ describe('parser (exif data)', () => {
 
 		})
 
+		describe('FLIR DUO PRO 640 R 19mm', () => {
+
+			it('RGB', async () => {
+				const exif = await parse(buffers['FLIR_DUO_PRO_640_R_19mm.JPG'], options)
+				assert.exists(exif, `exif doesn't exist`)
+				assert.strictEqual(exif.Make, 'FLIR')
+				assert.strictEqual(exif.Model, 'Duo Pro R')
+				assert.strictEqual(exif.XResolution, 1)
+				assert.strictEqual(exif.YResolution, 1)
+				assert.strictEqual(exif.ResolutionUnit, 1)
+				assert.strictEqual(exif.YCbCrPositioning, 1)
+				assert.strictEqual(exif.CameraSerialNumber, '8340254')
+				assert.strictEqual(exif.ExifVersion, '0231')
+				assert.strictEqual(exif.ComponentsConfiguration, 'Y, Cb, Cr, -')
+				assert.strictEqual(exif.FocalLength, 19)
+				assert.strictEqual(exif.FlashpixVersion, '0100')
+				assert.strictEqual(exif.ColorSpace, 65535)
+				assert.strictEqual(exif.FocalPlaneXResolution, 10.88)
+				assert.strictEqual(exif.FocalPlaneYResolution, 8.704)
+				assert.strictEqual(exif.FocalPlaneResolutionUnit, 4)
+				assert.strictEqual(exif.GPSVersionID, '2.2.0.0')
+				assert.strictEqual(exif.GPSLatitudeRef, 'N')
+				assert.deepEqual(exif.GPSLatitude, [ 51, 23, 21.55 ])
+				assert.strictEqual(exif.GPSLongitudeRef, 'E')
+				assert.deepEqual(exif.GPSLongitude, [ 12, 30, 40.62 ])
+				assert.strictEqual(exif.GPSAltitudeRef, 0)
+				assert.strictEqual(exif.GPSAltitude, 170.6)
+				assert.strictEqual(exif.GPSTimeStamp, '11:45:50')
+				assert.strictEqual(exif.GPSSpeedRef, 'K')
+				assert.strictEqual(exif.GPSSpeed, 0.52)
+				assert.strictEqual(exif.GPSTrackRef, 'T')
+				assert.strictEqual(exif.GPSTrack, 281.62)
+				assert.strictEqual(exif.GPSMapDatum, 'WGS84')
+				assert.strictEqual(exif.latitude, 51.389319444444446)
+				assert.strictEqual(exif.longitude, 12.511283333333333)
+
+                // XMP
```

Comment:
I guess I messed up here? :P

Reply:
I will ident the comment once Oli took a look as well to the PR :)

Reply:
![image](https://user-images.githubusercontent.com/17416198/95055780-117b1d00-06f4-11eb-94e7-0f59d5b6ce29.png)
seems fine in webstorm :p

Reply:
I think it's because we use spaces and they use tabs?